config/locales/simple_form.ca.yml

```diff
@@ -4,53 +4,64 @@ ca:
     hints:
       defaults:
         avatar: PNG, GIF o JPG. Màxim 2MB. Serà escalat a 120x120px
+        digest: S'envia després d'un llarg període d'inactivitat amb un resum de les mencions que has rebut en la teva absència
         display_name:
-          one: Queda <span class="name-counter">1</span> caràcter
-          other: Queden <span class="name-counter">%{count}</span> caràcters
+          one: <span class="name-counter">1</span> character left
+          other: <span class="name-counter">%{count}</span> characters left
```

Comment:
Translation reverted by merging. Is it okay @salvadorpla ?

Reply:
Yes, it's okay. Had some issues with git.